docs/voicemail-how-to.md

```diff
@@ -97,7 +99,11 @@ fun playVoicemail(voicemailId: String) {
 
     val tempFile = createTempFile(prefix = "voicemail", suffix = ".mp3")
     val outputStream = tempFile.outputStream()
-    voicemail.voicemailFile.writeTo(outputStream)
+    if (voicemail.bytesCase != BytesCase.WAV) {
```

Comment:
This doesn't make sense to me, why allow for different bytesCases if there aren't any?

Reply:
At the moment there aren't any other so I used an `if` rather than a `when`. The point of having it in the first place was to be future proof so that we can deliver several formats later without breaking changes.
I can use a `when` if you'd prefer, to show that there might be more in the future.

Reply:
I would prefer a `when`, but I'm still surprised that any of the formats can be unexpected. Could you explain that part (on Slack maybe)?

Reply:
I updated this with a when